scanners/pshtt.py

```diff
@@ -120,14 +121,17 @@ def to_rows(data):
 
 
 headers = [
-    "Canonical URL", "Live", "Redirect", "Redirect To",
+    "Canonical URL", "Live", "HTTPS Full Connection", "HTTPS Client Auth Required",
+    "Redirect", "Redirect To",
```

Comment:
I definitely would want to move `HTTPS Full Connection` and `HTTPS Client Auth Required` later in the column list, since the several columns (canonical URL, redirect destination) are very useful to keep together as the sort of introduction to a row, to observe behavior at a glance.